tests/dogtag/pytest-ansible/provision/osp_vars.py

```diff
@@ -0,0 +1,10 @@
+OS_AUTH_URL = "https://rhos-d.infra.prod.upshift.rdu2.redhat.com:13000/v3"
+OS_PROJECT_NAME = "certsys-jenkins"
+OS_USERNAME = "psi-certsys-jenkins"
+OS_REGION_NAME = "regionOne"
+OS_TENANT_NAME = "certsys-jenkins"
+OS_KEYPAIR_NAME = "certsys-jenkins3"
+OS_NETWORK_ID = ['25ec4907-36fc-4035-b8d5-b797246330f2', '5058fef2-f89f-4e70-9e01-66af2847ddc4',
+                 '60cacaff-86a6-4f88-82a4-ed3023724df1']
```

Comment:
Are these values referring to a specific environment? Could we get this from environment variable?

Reply:
Done changes. !